test/gtest/common/test_helpers.cc

```diff
@@ -80,6 +81,48 @@ void safe_usleep(double usec) {
     }
 }
 
+void print_ip(char *if_name, struct sockaddr *ifa_addr)
```

Comment:
const* for both

Reply:
also, better have a function which returns std::string with the address - would be more generic

Reply:
you mean that would only call ucs_sockaddr_str()?

Reply:
yes